tests/src/Traits/ContextualLinksTrait.php

```diff
@@ -83,7 +83,7 @@ protected function findContextualLinkButton(NodeElement $element): NodeElement {
     // Check if the wrapper for the contextual links is present on the page.
     // Since the button is appended by the contextual.js script, we might need
     // to wait a bit before the button itself is visible.
-    $button = $element->waitFor(5, function (NodeElement $element): ?NodeElement {
```

Comment:
How come we need such a big increase? It shouldn't take this much to render the links...

Reply:
I think this was increased by @claudiu-cristea since in this PR a cache clear was added in the `featured_entities` test to temporarily work around an unknown issue. One of the steps following this cache clear is involving contextual links. I'm guessing it might take more than 5 seconds on test infrastructure to generate the contextual links on a cold cache. This is not surprising given the fact that they are generated in a separate request.

I'm cool with this change. It reduces the chance of random failures, and it doesn't impact the test speed at all, since this `ElementInterface::waitFor()` method is polling every 100ms and will exit as soon as the condition is met. Only when this exact step will fail, the test runner will wait 30 seconds instead of 5 before failing the test. This should only occur during development so it is acceptable for me.

I'm open to change this to another value though, but this is not blocking the PR for me.